recommender/commands/train.py

```diff
@@ -42,7 +43,8 @@ def _rl_v2():
 
 
 def _all():
```

Comment:
maybe we can add a warning to the cli user (devloper): "callling this will delete all of the existing ML models, do you agree (y/n) ?

Reply:
Good remark. I have added that